comments.php

```diff
@@ -37,9 +36,11 @@
 		);
 
 		?>
+		<?php
+			the_comments_pagination();
```

Comment:
I would prefer if we implemented [Foundation pagination](https://foundation.zurb.com/sites/docs/pagination.html) styles using custom SCSS classes here. Can this be done with that function? Or do we need to fix `Foundationpress_Comments` so it paginates?

Reply:
You are right. I will update the pull request. I will replace the `the_comments_pagination();` funktion with a Foundation specific one. Similar to the `foundationpress_pagination()` function.